src/buildkite_test_collector/collector/run_env.py

```diff
@@ -3,8 +3,8 @@
 from dataclasses import dataclass
 from typing import Dict, Optional
 from uuid import uuid4
-
 import os
+from .constants import COLLECTOR_NAME, VERSION # pylint: disable=W0611
```

Comment:
Python linter doesn't recognise that imports that are interpolated _are_ actually used, so disabling for this line
https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unused-import.html